.github/workflows/tests.yml

```diff
@@ -1,5 +1,11 @@
 name: Tests
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
 
```

Comment:
We need to run tests on all PRs

Reply:
Yeah . This make sense.

Reply:
The block should look like this then

```
on:
  push:
    branches:
      - master
  pull_request:
```

Right now it's only running PRs on branches that are merging with `master` I believe. Tests didn't run on this PR yet.